client/src/components/card/Category.jsx

```diff
@@ -30,15 +30,41 @@ const Category = () => {
   }, [header]);
 
   const getColumnsForRow = () => {
-    let items = headers.map((header, index) => {
+    let items = headers.map((header) => {
```

Comment:
I would destructure the variables from `header`, so you don't have to access everything like `header.image` or `header.id` each time.

```suggestion
    let items = headers.map(({ id, image, name, episode, season, price }) => {
```